code/game/machinery/vending.dm

```diff
@@ -1225,3 +1225,30 @@
 					/obj/item/weapon/reagent_containers/food/snacks/old/hotdog = 1,
 					/obj/item/weapon/reagent_containers/food/snacks/old/taco = 1
 					)
+
+/obj/machinery/vending/whitedragon
+	name = "White Dragon"
+	desc = "A vendor shining with bright lamps, it appears to be selling oriental food."
```

Comment:
Oriental is generally considered a slur, at least where I'm from. I'd say switch it to Asian, although right now it only seems to be Japanese food, so maybe just change it to that?

Reply:
Yikes, unintentional. Ported it from a PR I made on an old dead server, I'll change it to that.